### PR TITLE
[enterprise-4.4] Added note per QE

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.4
Waiting QE approval: @miabbott  or @bgilbert 
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1834830

### CODE REVIEW

File: modules/installation-user-infra-machines-pxe.adoc [hunk 1]
```diff
@@ -139,6 +139,11 @@ file on your TFTP server. The `coreos.inst.image_url` parameter value is the
 location of the compressed metal RAW image on your HTTP server, and the
 `coreos.inst.ignition_url` parameter value is the location of the bootstrap
 Ignition config file on your HTTP server.
++
+[NOTE]
+====
+You can add use serial console by appending `console=ttyS0`.
```
Comment (ahardin-rh) [inline modules/installation-user-infra-machines-pxe.adoc:145]: You can use serial console  (same for the other instances in this PR)

**Comment by ahardin-rh** [comment]
Just one suggestion. Otherwise, looks good!

File: modules/installation-user-infra-machines-pxe.adoc (hunk 1 above)
Comment (sagidlow) [reply modules/installation-user-infra-machines-pxe.adoc]: Updated!

File: modules/installation-user-infra-machines-pxe.adoc [hunk 2]
```diff
@@ -139,6 +139,11 @@ file on your TFTP server. The `coreos.inst.image_url` parameter value is the
 location of the compressed metal RAW image on your HTTP server, and the
 `coreos.inst.ignition_url` parameter value is the location of the bootstrap
 Ignition config file on your HTTP server.
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`.
```
Comment (bgilbert) [inline modules/installation-user-infra-machines-pxe.adoc:145]: It'd be good to spell this out in a bit more detail.  Perhaps something like this:

```suggestion
This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```

**Review by bgilbert — COMMENTED** [review]
For 4.4 and 4.5, we should also remove `console=tty0` and `console=ttyS0` arguments from the default configuration.

**Comment by sagidlow** [comment]
@bgilbert  please verify the changes per your request.

File: modules/installation-user-infra-machines-pxe.adoc [hunk 3]
```diff
@@ -158,6 +163,12 @@ parameter value is the location of the bootstrap Ignition config file.
 For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 <3> Specify the location of the `initramfs` file that you uploaded to your HTTP
 server.
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```
Comment (miabbott) [inline modules/installation-user-infra-machines-pxe.adoc:169]: There is no `APPEND` line for the iPXE case.

```suggestion
To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `kernel` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```

File: modules/machine-user-infra-machines-pxe.adoc [hunk 4]
```diff
@@ -26,21 +26,29 @@ TIMEOUT 20
...
 <2> If you use multiple NICs, specify a single interface in the `ip` option. For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 <3> Specify the location of the `initramfs` file that you uploaded to your HTTP server.
-
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```
Comment (miabbott) [inline modules/machine-user-infra-machines-pxe.adoc:52]:
```suggestion
To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `kernel` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 5]
```diff
@@ -136,11 +136,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
 sh-4.2# cat /host/proc/cmdline
-BOOT_IMAGE=/ostree/rhcos-... console=tty0 console=ttyS0,115200n8
+BOOT_IMAGE=/ostree/rhcos-... ,115200n8
```
Comment (miabbott) [inline modules/nodes-nodes-kernel-arguments.adoc:139]: I don't think this needs to be changed; these are baked in default values in our images

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 6]
```diff
@@ -136,11 +136,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
...
 
 sh-4.2# exit
 ----
 +
 You should see the `selinux=0` argument added to the other kernel arguments.
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
+====
```
Comment (miabbott) [inline modules/nodes-nodes-kernel-arguments.adoc:151]: This is isn't applicable to these docs.

**Comment by bgilbert** [comment]
I concur with Micah's comments, and would also note https://github.com/openshift/openshift-docs/pull/29587#discussion_r578966550.

**Review by bgilbert — APPROVED** [review]

**Pull request merged**